src/todo-item.ts

```diff
@@ -39,7 +39,7 @@ class TodoItemElement extends HTMLElement {
         this.dispatchEvent(new CustomEvent("checked", {
             detail: {
                 index: this.index,
-                checked: ev.target.checked
+                checked: ev.detail.checked
```

Comment:
In this case `ev` is actualy not a CustomEvent, so it does not have a `detail` field and makes it generate an error.
I will update the type in here.